document/js-api/index.bs

```diff
@@ -615,7 +615,7 @@ interface Table {
 <div>
 A {{Table}} object represents a single [=table instance=]
 which can be simultaneously referenced by multiple {{Instance}} objects. Each
-{{Table}} object has one internal slots:
+{{Table}} object has two internal slots:
```

Comment:
"has the following internal slots" so it'll always be right?

Reply:
True, probably should change the other occurrences in the file if we go that route.